addins/Cake.ServiceOrchestration.yml

```diff
@@ -0,0 +1,13 @@
+Name: Cake.ServiceOrchestration
```

Comment:
Should be filtered as it is not an addin

Reply:
Done. Cake.ServiceOrchestration added to blacklist